.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ethernet_controller/import/basejump_stl"]
```

Comment:
I think we want to reference out of tree like here: https://github.com/black-parrot-hdk/black-parrot-subsystems/blob/760a888eb8b72bdd6972abf2366deb20b0996ccf/axi/test/bsg_axil_demux/Makefile#L5

Reply:
So we don't need import/ anymore?

Reply:
Thinking of the PULP use in the future we might need import/